cmd/server/pactasrv/pacta_version.go

```diff
@@ -47,6 +47,13 @@ func (s *Server) ListPactaVersions(ctx context.Context, request api.ListPactaVer
 // Creates a PACTA version
 // (POST /pacta-versions)
 func (s *Server) CreatePactaVersion(ctx context.Context, request api.CreatePactaVersionRequestObject) (api.CreatePactaVersionResponseObject, error) {
+	if err := anyError(
+		checkStringLimitSmall("name", request.Body.Name),
+		checkStringLimitMedium("digest", request.Body.Digest),
```

Comment:
If the digest is a SHA256 value, it'll be a fixed size, which is much smaller than 1000 ASCII characters/bytes, meaning this could be `Small`, and same elsewhere

Reply:
Great call. Done.